### ISSUE TITLE
The "configuration" object is not available

### ISSUE THREAD

**Issue by abix5** [mooltiverse/nyx#137]
I tried using the [extended](https://raw.githubusercontent.com/mooltiverse/nyx/master/docs/_includes/.nyx-extended.yaml) configuration template without changes in the master branch:

`time="2023-01-23T17:26:40Z" level=debug msg="evaluating tag 'v0.3.0': the selected release type specifies an additional filter '^()?([0-9]\\d*)\\.([0-9]\\d*)\\.([0-9]\\d*)$' (after template evaluation) for tags but the tag doesn't match it so it will be ignored. The tag is applied to commit 'db30d5db5278ac85085c0428c9d510df1b1c977c'"`

The regular expression is there — `^({{configuration.releasePrefix}})?([0-9]\\d*)\\.([0-9]\\d*)\\.([0-9]\\d*)$`

I tried different options, added `releasePrefix` to `.nyx.yml`, tried to access other variables of the `configuration` object. Everything is useless, maybe I'm doing something wrong?

[1.3.1-hotfix130.2](https://github.com/mooltiverse/nyx/releases/tag/1.3.1-hotfix130.2)

**Comment by flelli** [comment]
Hi @abix5 ,

let me ask a few questions to better understand:

1. are you using the Java (Gradle) or Go (command line) version?
2. the configuration file attached is named `.nyx-extended.yaml` and it then references `.nyx.yaml` (`configurationFile: ".nyx.yaml"`) so I guess you're explicitly loading `.nyx-extended.yaml` (i.e. by using the [`configurationFile`](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/global-options/#configuration-file) option on the command line) and your intention is to *compose* the configuration from the two files, right? If so, `.nyx.yaml` won't be loaded unless you use one of the two as a [shared configuration file](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/global-options/#shared-configuration-file) (depending on the priority you want to give to the two files)
3. have you tried dumping the [state file](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/global-options/#state-file) and inspected its contents?

If the above questions don't help, please share a [debug log](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/global-options/#verbosity) and a [state file](https://mooltiverse.github.io/nyx/guide/user/configuration-reference/global-options/#state-file)

**Comment by abix5** [comment]
Hi @flelli,
1. Go (command line) version
2. The project contains a .nyx.yml file
```
changelog:
...
git:
...
commitMessageConventions: 
...
initialVersion: "0.1.0"
releaseLenient: true
releasePrefix: "v"
releaseTypes:
  enabled: 
    - mainline
    - integration
  publicationServices:
    - gitlab
  remoteRepositories:
    - origin
  items: 
    mainline: 
...
    integration: 
      collapseVersions: false
      description: "{{#fileContent}}CHANGELOG.md{{/fileContent}}"
      filterTags: "^({{configuration.releasePrefix}})?([0-9]\\d*)\\.([0-9]\\d*)\\.([0-9]\\d*)$"
      gitCommit: "false"
      gitPush: "true"
      gitTag: "true"
      gitTagMessage: "Tag version {{version}}"
      matchBranches: "^(dev)$"
      matchEnvironmentVariables: 
        CI: "^true$"
      matchWorkspaceStatus: "CLEAN"
      publish: "true"
      versionRangeFromBranchName: false
scheme: "SEMVER"
resume: false
services:
  gitlab: 
...
stateFile: ".nyx-state.yml"
verbosity: DEBUG
```
3. The repository does not contain a status file. Gitlab's job is always executed on a clean repository.

And the value in {{configuration.releasePrefix}} is always empty

**Comment by flelli** [comment]
Thanks @abix5 , can ypu please share the debug log and the state file?
Or, if it's a public repo, can you share the link where I can clone it and try?

**Comment by abix5** [comment]
@flelli, Thank you very much. I just created a repository.
[Public repo](https://github.com/abix5/auto-release-test)

**Comment by flelli** [comment]
Bug confirmed. I'll issue an hotfix shortly

**Comment by flelli** [comment]
The fix is available in version [1.3.4](https://github.com/mooltiverse/nyx/releases/tag/1.3.4).
